solution/networking/multi-account-multi-vpc/versions.tf

```
@@ -0,0 +1,9 @@
+terraform {
+  required_providers {
+    alicloud = {
+      source = "hashicorp/alicloud"
+      version = ">= 1.96.0"
```

```suggestion
      version = ">= 1.114.0"
```

1.96.0 还不支持 Enhanced 的 NAT Gateway